indico/modules/events/abstracts/forms.py

```diff
@@ -200,6 +227,34 @@ def __init__(self, *args, **kwargs):
         self.merged_into.excluded_abstract_ids = {abstract.id}
 
 
+class AbstractReviewForm(IndicoForm):
+    """Form for reviewing an abstract"""
+
+    _order = ('comment', 'proposed_action', 'proposed_contribution_type', 'proposed_related_abstract')
+
+    comment = TextAreaField(_("Comments"))
+    proposed_action = IndicoEnumSelectField(
+        _("Proposed Action"), [DataRequired()],
+        enum=AbstractAction, skip={AbstractAction.change_tracks})
```

Comment:
Mixing indentation types. Any reason we don't go for good ol' aligning to opening delimiter here?